hello-world-1/pom.xml

```diff
@@ -28,14 +112,22 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
+<version>4.12</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest-all</artifactId>
-            <version>${version.hamcrest}</version>
+            <version>1.3</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>ae.teletronics.nlp</groupId>
+            <artifactId>categorisation</artifactId>
+            <version>1.7-SNAPSHOT</version>
```

Comment:
We cannot depend on snapshots. Could you please set this to the closest release version?

What is this dependency for?

Reply:
It appears Intellij automatically added this depedency. We can remove this

Reply:
@dinesh19aug :+1: 

Okay! Could you please remove it?

Reply:
I will remove and create a PR

Reply:
@dinesh19aug ,

There is already a pull request for this, as far as I recall.

The problem is not so much in the PR, but in the fact that these results are inconclusive. We'd already reached these results. Our numbers ought to be much, much higher. We still need to figure out what's going on.

Reply:
@dinesh19aug I'm confused... You already have a PR? :D